CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ This CHANGELOG follows the format located [here](https://github.com/sensu-plugin
 
 ## [Unreleased]
 
+### Breaking
+
+- Moved to Chocolatey to manage sensuctl and sensu-agent on Windows. Previously, this installed version 6.1.0 for Windows. This will now install the latest version if the `version` attribute is not set for `sensu_agent`. This may also cause issues with existing installations, as the Chocolatey package manager cannot see the existing installation. (@kovukono)
```

Comment:
I think there are a couple things to unpack here let's see if we can break this up and expand a little:
- install method changed: moved from downloaded msi from sensu repos to chocolatey.
- install interface changed: attribute to resource property. See "how it affects users"
- who this affects: windows users, possibly already installed setups, etc?
- how it affects users: for new installs updating the attribute to the resource property. What about installs that are already present? Should we have a `:migrate` method that can do some extra bits to either register it or uninstall/reinstall? What does this look like? Even if we can't provide an automated migration we need to help explain to users what they might need to do first. For example they might need to do the equivalent of (yes its pseudo and not really something that probably makes sense in windows I just don't know much about windows management these days) a `knife ssh 'roles:base' 'sudo uninstall-sensu-agent'` first so that chef can be ready to drop everything in place for going forward.

Reply:
We could make a migrate for the sensuctl, but the agent is installing in the same directory. It looks like it should be fine.

Reply:
I went and added a migrate method for removal, and updated the readme to reference how to use it. I also broke down the Changelog a bit more. I'm honestly not sure how the `knife windows` plugin would function for this, but the knife command to remove the old path from the existing path probably would be fairly clunky, something along the lines of `knife winrm "recipes:your-sensu-go-wrapper\\:\\:recipe" 'powershell.exe "$path = [System.Environment]::GetEnvironmentVariable(\'PATH\',  \'System\'); $path = ($path.Split(\';\') | Where-Object { $_ -ne \'c:\Program Files\Sensu\sensu-cli\bin\sensuctl\' }) -join \';\'; [System.Environment]::SetEnvironmentVariable(\'PATH\', $path, \'System\')"'`.